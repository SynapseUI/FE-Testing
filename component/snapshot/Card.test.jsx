import React from 'react';
import { render } from '@testing-library/react';
import UserCard from './Card';


const exampleUser = {
  name: 'Zoe Lee',
  age: 25,
  gender: 'Female',
  country: 'Korea'
};

it('matches snapshot', () => {
  const { asFragment } = render(<UserCard user={exampleUser} />);

  expect(asFragment()).toMatchSnapshot();
});
