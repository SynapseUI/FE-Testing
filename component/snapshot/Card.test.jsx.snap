// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="user-card"
    data-testid="Zoe Lee-card"
  >
    <h3>
      Zoe Lee
    </h3>
    <hr />
    <ul
      class="user-info-list"
    >
      <li
        class="user-info-item"
      >
        <h4>
          Age:
        </h4>
        <p>
          25
        </p>
      </li>
      <li
        class="user-info-item"
      >
        <h4>
          Gender:
        </h4>
        <p>
          Female
        </p>
      </li>
      <li
        class="user-info-item"
      >
        <h4>
          Country:
        </h4>
        <p>
          Korea
        </p>
      </li>
    </ul>
  </div>
</DocumentFragment>
`;
